configs/mender_convert_config

```diff
@@ -157,6 +157,14 @@ MENDER_ADDON_CONFIGURE_INSTALL="n"
 # Valid values are "latest" (default), "master" or a specific version
 MENDER_ADDON_CONFIGURE_VERSION="latest"
 
+# Directory containing deb packages to be installed in the image
+#
+MENDER_INSTALL_INPUT_PACKAGES_PATH="input/deb"
+
+# Install deb packages located in MENDER_INSTALL_INPUT_PACKAGES_PATH
+#
+MENDER_INSTALL_INPUT_PACKAGES="n"
+
```

Comment:
Thinking out loud: should we call these variables something else by Mender? Like `USER_INSTALL_INPUT_PACKAGES_PATH`, or just `INSTALL_PACKAGES_PATH`. The reason being that this is a method for install any package.

If so, we should also move it somewhere else that it is not next to the other MENDER variables.

Reply:
Hi @fcervigni-EP and thanks for your input. Highly appreciated. I don't have an strong opinion, I was thinking out loud. It is totally fine with me to leave the `MENDER_` prefix. Do you agree @danielskinstad?

Overall I agree with your principle, that a tool should stick to the same prefix everywhere to avoid environment poisoning of any sort. However even in this file we have not been consistent, as they are other variables that don't have the prefix. Nevertheless it is a good idea to keep using `MENDER_` as much as possible in the new features :+1:

Reply:
Yes, I agree :) I will keep using `MENDER_`.